driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideEncryptionAwsCredentialFromEnvironmentTest.java

```diff
@@ -192,6 +196,60 @@ public void shouldThrowMongoConfigurationIfSupplierReturnsDoesSomethingUnexpecte
         }
     }
 
+    @Test
+    public void shouldThrowMongoCryptIfNamedKmsProviderHasEmptyParametersInEncryptionSettings() {
+        assumeTrue(serverVersionAtLeast(4, 2));
+        assumeFalse(System.getenv().containsKey("AWS_ACCESS_KEY_ID"));
+        assumeTrue(isClientSideEncryptionTest());
```

Comment:
Unless I am crazy, these two assumptions contradict each other, and can never be both satisfied: the first one requires that the environment does not have the `AWS_ACCESS_KEY_ID` variable, while the second one requires the value of the `AWS_ACCESS_KEY_ID` variable to be non-empty.

Reply:
Good catch! Removed `assumeFalse(System.getenv().containsKey("AWS_ACCESS_KEY_ID"));`, thanks!